data/tags.json

```diff
@@ -54,38 +54,38 @@
       "more info on Matrix: <https://matrix.org/about/>, <https://matrix.org/bridges/>"
     ]
   },
-  { "name": "contr",
-    "aliases": [],
+  { "name": "contribute",
+    "aliases": ["contr", "contrib"],
     "content": [
       "**Contributing to Termux**",
-      "-How you can contribute to the Termux project:",
+      "- How you can contribute to the Termux project:",
       "<https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md>",
-      "-How to build packages:",
+      "- How to build packages:",
       "<https://github.com/termux/termux-packages/wiki/Building-packages>",
-      "-Differences from Linux:",
+      "- Differences from Linux:",
       "<https://wiki.termux.com/wiki/Differences_from_Linux>"
     ]
   },
   { "name": "emacs4newcomers",
```

Comment:
nit: Let's use `emacs` as the name and `emacs4newcomers` as an alias. I don't think I've ever seen anyone type out 4newcomers.